docs/contributor-guide/developer-guide/deploy-system/prerequisits.md

```diff
@@ -3,12 +3,256 @@
+    - IAM role name: leave the default of 'OrganizationAccountAccessRole'.
+![Add AWS account details](../../../.gitbook/assets/aws-add-dev-account-details.png)
+
+4. Account creation will take a minute or two. Once it is created, you will see it listed on the [AWS Organizations page](https://us-east-1.console.aws.amazon.com/organizations/v2/home/root).
+
+5. Once the account is created, copy the account number somewhere for yourself temporarily. You will need it to log in to the new account and configure it.
+![Newly created AWS account in Organizational structure list](../../../.gitbook/assets/aws-new-dev-account-in-list.png)
+
+6. (For Exygy employees only) Move the newly created dev account so that it is nested inside the `civiform-dev` folder via the following steps:
+    - Select the checkbox for the new account.
+    - Click the "Actions" dropdown.
+    - Select "Move" under the "AWS account" section in the dropdown.
+![Move newly created AWS account - part 1](../../../.gitbook/assets/aws-move-dev-account-1.png)
+    - On the following screen, select the `civiform-dev` folder and then click the "Move AWS account" button.
+![Move newly created AWS account - part 2](../../../.gitbook/assets/aws-move-dev-account-2.png)
+
+### Log in to your new dev AWS Account
+
+1. From any screen in the AWS Console, click the drop down in the upper right hand corner and click "Switch role". 
+![Switch role dropdown](../../../.gitbook/assets/aws-switch-role-1.png)
+
+2. Click "Switch Role" on the next screen.
+![Switch role screen](../../../.gitbook/assets/aws-switch-role-2.png)
+
+3. Input the following details and click "Switch Role":
+    - Account: Input the account number for your new dev account (copied in step 5 above).
+    - Role: Input "OrganizationAccountAccessRole"
+    - Display Name: Input the dev account name you created when making the account (e.g `jdoe-dev`)
+![Switch role details](../../../.gitbook/assets/aws-switch-role-3.png)
+
+4. Verify that you are now viewing the newly created account by navigating to the [IAM dashboard](https://us-east-1.console.aws.amazon.com/iamv2/home?region=us-east-1#/home) and making sure that the Account ID listed in the sidebar on the right matches the account ID for your newly created dev account.
+![Check new dev account number](../../../.gitbook/assets/aws-confirm-switched-roles.png)
+
+Note: You only have to log in to your dev account one time. After that, it will be listed under "Role history" in the drop down in the upper right and you will be able to easily toggle back and forth between accounts by clicking on the role in that menu.
+
+![Role history in dropdown](../../../.gitbook/assets/aws-role-history.png)
+
+### Configure new AWS Account
+
+1. Navigate to your newly created dev account's [IAM page](https://us-east-1.console.aws.amazon.com/iamv2/home?region=us-east-1#/home).
+
+2. Click the "Create" button next to the Account Number on the righthand sidebar.
```

Comment:
Maybe `Click the "Create" button under `Account Alias` on the righthand sidebar`?